Change default preferred-dark-theme to `navy`. (#1199)
rust-lang/mdBook: book-example/src/format/theme/README.md

- ***book.js*** is mostly used to add client side functionality, like hiding /
  un-hiding the sidebar, changing the theme, ...
- ***highlight.js*** is the JavaScript that is used to highlight code snippets,

>   you should not need to modify this.

Reviewer: ```suggestion
  you do not need to modify this.
```

template and only add / modify what you need. You can copy the default theme
into your source directory automatically by using `mdbook init --theme` just
remove the files you don't want to override.

> If you completely replace all built-in themes, be sure to also set

Reviewer: ```suggestion
If you completely replace all the built-in themes, ensure that [`output.html.preferred-dark-theme`] 
```

> [`output.html.preferred-dark-theme`] in the config, which defaults to the

Reviewer: ```suggestion
is set in the config, which defaults to the
```
